mLRS/CommonRx/mavlink_interface_rx.h

```diff
@@ -132,6 +133,7 @@ void MavlinkBase::Init(void)
     radio_status_txbuf = 0;
     txbuf_state = TXBUF_STATE_NORMAL;
 
+    parsing_in_cnt = 0;
```

Comment:
maybe in analogy bytes_parser_in_cnt ?

Reply:
Sure, if you prefer, though I think what I have is at least as clear.